### PR TITLE
Added CacheTask

### PR DESCRIPTION
**Pull request by gallynaut**

### CODE REVIEW

File: libraries/protos/job_schemas.proto [hunk 1]
```diff
@@ -365,13 +367,32 @@ message OracleJob {
...
+    //   repeated CacheItem parse = 2;
+    // }
+    // message CacheItem {
+    //   optional string variable_name = 1;
+    //   repeated Task parse_tasks = 2;
+    // }
+    // repeated CacheSource cache_sources = 1;
+    message CacheItem {
+      optional string variable_name = 1;
+      repeated Task tasks = 2;
```
Comment (mgild) [inline libraries/protos/job_schemas.proto:389]: Should we make this an OracleJob?

Comment (jessupjn) [reply libraries/protos/job_schemas.proto:389]: +1

File: libraries/protos/job_schemas.proto [hunk 2]
```diff
@@ -90,6 +89,8 @@ message OracleJob {
       double value = 1;
       // Specifies an aggregatorr to pull the value of.
       string aggregator_pubkey = 2;
+      // A stringified big.js or variable name.
```
Comment (mgild) [inline libraries/protos/job_schemas.proto:92]: I would say `A stringified big.js. Accepts variable expansion syntax.`

File: libraries/protos/job_schemas.proto [hunk 3]
```diff
@@ -126,11 +127,12 @@ message OracleJob {
       string aggregator_pubkey = 2;
       // A job whose result is computed before dividing our numerical input by that result.
       OracleJob job = 3;
+      // A stringified big.js or variable name.
+      string big = 4;
```
Comment (mgild) [inline libraries/protos/job_schemas.proto:131]: I would say A stringified big.js. `Accepts variable expansion syntax.`

File: libraries/protos/job_schemas.proto [hunk 4]
```diff
@@ -139,6 +141,8 @@ message OracleJob {
       string aggregator_pubkey = 2;
       // A job whose result is computed before multiplying our numerical input by that result.
       OracleJob job = 3;
+      // A stringified big.js or variable name.
+      string big = 4;
```
Comment (mgild) [inline libraries/protos/job_schemas.proto:145]: ^

**Review by mgild — APPROVED** [review]

**Pull request merged**